packages/material-ui/src/ButtonBase/ButtonBase.js

```diff
@@ -326,7 +336,7 @@ ButtonBase.propTypes = {
   action: PropTypes.func,
   /**
    * Use that property to pass a ref callback to the native button component.
-   * @deprecated
+   * @deprecated Use `ref` instead
```

Comment:
So, we gonna use the `deprecatedPropType()` helper in v4.1?

Reply:
That's the plan. The existence of this prop doesn't actually block us so we might as well defer a breaking change here so that migration is easier. I still have the typography v2 in mind and while the warnings were just to much in that case it is still considered something you have to address when upgrading.